### PR TITLE
Fix doller sign escaping issue

### PR DESCRIPTION
**Pull request by arnaudsjs**
# Description

Fix doller sign escaping issue in installation documentation.

# Self Check:

- [ ] ~~Attached issue to pull request~~
- [x] Changelog entry
- [ ] ~~Type annotations are present~~
- [ ] ~~Code is clear and sufficiently documented~~
- [ ] ~~No (preventable) type errors (check using make mypy or make mypy-diff)~~
- [ ] ~~Sufficient test cases (reproduces the bug/tests the requested feature)~~
- [x] Correct, in line with design
- [ ] ~~End user documentation is included or an issue is created for end-user documentation~~

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: changelogs/unreleased/fix-escaping-issue-in-install-docs.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+description: Fix dollar sign escaping issue in installation documentation
+change-type: patch
+destination-branches: [master]
```
Comment (sanderr) [inline changelogs/unreleased/fix-escaping-issue-in-install-docs.yml:3]:
```suggestion
destination-branches: [master, iso4]
```
Or am I missing something?

**Review by sanderr — CHANGES_REQUESTED** [review]
What about the one on line 43?

File: changelogs/unreleased/fix-escaping-issue-in-install-docs.yml (hunk 1 above)
Comment (arnaudsjs) [reply changelogs/unreleased/fix-escaping-issue-in-install-docs.yml:3]: This is the installation documentation for inmanta OSS, so `master` should be sufficient.

Comment (sanderr) [reply changelogs/unreleased/fix-escaping-issue-in-install-docs.yml:3]: I see it's present on the iso4 branch as well but I assume it gets replaced with the iso instructions then?

**Review by sanderr — APPROVED** [review]

File: changelogs/unreleased/fix-escaping-issue-in-install-docs.yml (hunk 1 above)
Comment (arnaudsjs) [reply changelogs/unreleased/fix-escaping-issue-in-install-docs.yml:3]: Yes, it is.

**Review by wouterdb — APPROVED** [review]

**Pull request closed without merging**